docs/extending/customising_group_views.md

```diff
@@ -106,3 +106,27 @@ INSTALLED_APPS = [
     ...,
 ]
 ```
+
+## Customising the Group Editor Object Permissions Order
+
+The order that object types appear in the Group Editor "Object permissions" and "Other permissions" sections can be configured by registering that order in one or more AppConfig definitions. The order value is typically an integer between 0 and 999, although this is not enforced.
+
+```python
+from django.apps import AppConfig
+
+
+class MyProjectAdminAppConfig(AppConfig):
+    name = "myproject_admin"
+    verbose_name = "My Project Admin"
+
+    def ready(self):
+        from wagtail.users.permission_order import register
+
+        register("gadgets.SprocketType", 150)
+        register("gadgets.ChainType", 151)
+        register("site_settings.Settings", 160)
```

Comment:
Should we use a kwarg here? I know this is not a hook but all other hooks with ordering take their order as a keyword arg. Just lets us add more options later if needed.

```suggestion
        register("site_settings.Settings", order=160)
```

Reply:
Changed in [eb0f1a5](https://github.com/wagtail/wagtail/pull/10469/commits/eb0f1a5aeccb835e7c0d75661cfbbb621d178447)